WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -90,6 +91,11 @@ - (NSString *)wdType
   return [FBElementTypeTransformer stringWithElementType:self.elementType];
 }
 
+- (NSString *)wdClass
+{
+  return FBTransferEmptyStringToNil(self.fb_class);
```

Comment:
what the point to transform it to nil here if this can be already done in the corresponding extension method?